decidim-core/lib/decidim/reminder_manifest.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This class acts as a manifest for reminders.
+  #
+  # This manifest is a simple object that holds and stores reminder
+  # and it's manager class.
+  class ReminderManifest
```

Comment:
😍 😍 

I know this is still in development, but this looks like it could be helpful for other kind of reminders like what's being made in #8551

Let me know when this is ready to be reviewed @lahdeero

Reply:
@andreslucena I have reviewed it and I was mostly happy about it. I did one renaming and added documentation.